diff --git a/common/export_map/Makefile.am b/common/export_map/Makefile.am
index 125988e..944bbf4 100644
--- a/common/export_map/Makefile.am
+++ b/common/export_map/Makefile.am
@@ -2,10 +2,3 @@ include $(top_srcdir)/common.mk
 
 noinst_LTLIBRARIES = libolaexportmap.la
 libolaexportmap_la_SOURCES = ExportMap.cpp
-
-TESTS = ExportMapTester
-check_PROGRAMS = $(TESTS)
-ExportMapTester_SOURCES = ExportMapTester.cpp ExportMapTest.cpp
-ExportMapTester_CXXFLAGS = $(CPPUNIT_CFLAGS)
-ExportMapTester_LDFLAGS = $(CPPUNIT_LIBS)
-ExportMapTester_LDADD = ./libolaexportmap.la ../utils/libolautils.la
diff --git a/common/logging/Makefile.am b/common/logging/Makefile.am
index 38f4474..e8118b7 100644
--- a/common/logging/Makefile.am
+++ b/common/logging/Makefile.am
@@ -2,11 +2,3 @@ include $(top_srcdir)/common.mk
 
 noinst_LTLIBRARIES = liblogging.la
 liblogging_la_SOURCES = Logging.cpp
-
-TESTS = LoggingTester
-check_PROGRAMS = $(TESTS)
-
-LoggingTester_SOURCES = LoggingTester.cpp LoggingTest.cpp
-LoggingTester_CXXFLAGS = $(CPPUNIT_CFLAGS)
-LoggingTester_LDFLAGS = $(CPPUNIT_LIBS)
-LoggingTester_LDADD = ./liblogging.la ../utils/libolautils.la
diff --git a/common/network/Makefile.am b/common/network/Makefile.am
index 2360c58..d153227 100644
--- a/common/network/Makefile.am
+++ b/common/network/Makefile.am
@@ -11,19 +11,3 @@ libolanetwork_la_SOURCES += PosixInterfacePicker.cpp
 endif
 
 EXTRA_DIST = PosixInterfacePicker.h WindowsInterfacePicker.h
-
-TESTS = NetworkTester
-check_PROGRAMS = $(TESTS)
-NetworkTester_SOURCES = SelectServerTester.cpp SocketTest.cpp \
-                        NetworkTest.cpp NetworkUtilsTest.cpp \
-                        InterfacePickerTest.cpp
-NetworkTester_CXXFLAGS = $(CPPUNIT_CFLAGS)
-NetworkTester_LDFLAGS = $(CPPUNIT_LIBS)
-NetworkTester_LDADD = ./libolanetwork.la \
-                      ../export_map/libolaexportmap.la \
-                      ../logging/liblogging.la \
-                      ../utils/libolautils.la
-
-if USING_WIN32
-  NetworkTester_LDFLAGS += -no-undefined -liphlpapi -lnetapi32 -lcap -lws2_32 -ldpnet -lwsock32
-endif
diff --git a/common/rdm/Makefile.am b/common/rdm/Makefile.am
index 21a5dfd..620706d 100644
--- a/common/rdm/Makefile.am
+++ b/common/rdm/Makefile.am
@@ -5,12 +5,3 @@ EXTRA_DIST = Pids.proto
 noinst_LTLIBRARIES = libolardm.la
 libolardm_la_SOURCES = RDMAPI.cpp RDMCommand.cpp RDMHelper.cpp \
                        QueueingRDMController.cpp
-
-TESTS = RDMTester
-check_PROGRAMS = $(TESTS)
-RDMTester_SOURCES = RDMAPITest.cpp RDMCommandTest.cpp RDMTester.cpp \
-                    QueueingRDMControllerTest.cpp UIDTest.cpp
-RDMTester_CPPFLAGS = $(CPPUNIT_CFLAGS) $(AM_CPPFLAGS)
-RDMTester_LDFLAGS = $(CPPUNIT_LIBS)
-RDMTester_LDADD = libolardm.la ../logging/liblogging.la \
-                  ../network/libolanetwork.la ../utils/libolautils.la
diff --git a/common/rpc/Makefile.am b/common/rpc/Makefile.am
index efd3af6..8160906 100644
--- a/common/rpc/Makefile.am
+++ b/common/rpc/Makefile.am
@@ -15,19 +15,6 @@ Rpc.pb.cc Rpc.pb.h: Rpc.proto
 TestService.pb.cc TestService.pb.h: TestService.proto
 	$(PROTOC) --cpp_out ./ TestService.proto
 
-TESTS = RpcTester
-check_PROGRAMS = $(TESTS)
-RpcTester_SOURCES = RpcTester.cpp RpcControllerTest.cpp\
-                    StreamRpcChannelTest.cpp StreamRpcHeaderTest.cpp
-nodist_RpcTester_SOURCES = TestService.pb.cc
-RpcTester_CXXFLAGS = $(CPPUNIT_CFLAGS)
-RpcTester_LDFLAGS = $(CPPUNIT_LIBS) -lprotobuf
-RpcTester_LDADD = ./libstreamrpcchannel.la \
-                  ../export_map/libolaexportmap.la \
-                  ../logging/liblogging.la \
-                  ../network/libolanetwork.la \
-                  ../utils/libolautils.la
-
 clean-local:
 	rm -f *.pb.{h,cc}
 
diff --git a/common/utils/Makefile.am b/common/utils/Makefile.am
index 5f055d5..ee9659b 100644
--- a/common/utils/Makefile.am
+++ b/common/utils/Makefile.am
@@ -3,13 +3,3 @@ include $(top_srcdir)/common.mk
 noinst_LTLIBRARIES = libolautils.la
 libolautils_la_SOURCES = ActionQueue.cpp DmxBuffer.cpp OlaThread.cpp \
                          RunLengthEncoder.cpp StringUtils.cpp TokenBucket.cpp
-
-TESTS = UtilsTester
-check_PROGRAMS = $(TESTS)
-UtilsTester_SOURCES = ActionQueueTest.cpp ClockTest.cpp CallbackTest.cpp \
-                      DmxBufferTest.cpp OlaThreadTest.cpp \
-                      RunLengthEncoderTest.cpp  StringUtilsTest.cpp \
-                      UtilsTester.cpp
-UtilsTester_CXXFLAGS = $(CPPUNIT_CFLAGS)
-UtilsTester_LDFLAGS = $(CPPUNIT_LIBS)
-UtilsTester_LDADD = libolautils.la ../logging/liblogging.la
diff --git a/common/web/Makefile.am b/common/web/Makefile.am
index c2dc8ff..684938b 100644
--- a/common/web/Makefile.am
+++ b/common/web/Makefile.am
@@ -2,10 +2,3 @@ include $(top_srcdir)/common.mk
 
 noinst_LTLIBRARIES = libolaweb.la
 libolaweb_la_SOURCES = JsonSections.cpp
-
-TESTS = WebTester
-check_PROGRAMS = $(TESTS)
-WebTester_SOURCES = JsonSectionsTest.cpp WebTester.cpp
-WebTester_CXXFLAGS = $(CPPUNIT_CFLAGS)
-WebTester_LDFLAGS = $(CPPUNIT_LIBS)
-WebTester_LDADD = libolaweb.la ../logging/liblogging.la ../utils/libolautils.la
diff --git a/configure.ac b/configure.ac
index b6c2b32..becfe49 100644
--- a/configure.ac
+++ b/configure.ac
@@ -243,12 +243,6 @@ AC_SUBST(pid_datadir)
 OLAD_DEFINES="-DHTTP_DATA_DIR=\\\"$www_datadir\\\" -DPLUGIN_DIR=\\\"$plugindir\\\" "
 AC_SUBST(OLAD_DEFINES)
 
-# check for cppunit
-AM_PATH_CPPUNIT(
-  1.9.6,
-  [],
-  [AC_MSG_ERROR([Missing cppunit, please install >= 1.9.6])])
-
 # gcov
 AC_ARG_ENABLE(gcov,
               AC_HELP_STRING([--enable-gcov],
diff --git a/ola/Makefile.am b/ola/Makefile.am
index d0e52bb..cea0dd0 100644
--- a/ola/Makefile.am
+++ b/ola/Makefile.am
@@ -14,19 +14,6 @@ pkginclude_HEADERS = $(HEADER_FILES)
 
 EXTRA_DIST = $(HEADER_FILES) OlaClientCore.h common-h.in
 
-
-# Test Programs
-TESTS = OlaClientTester
-check_PROGRAMS = $(TESTS)
-OlaClientTester_SOURCES = $(OLASERVER_SOURCES) OlaClientTester.cpp \
-                          StreamingClientTest.cpp
-OlaClientTester_CXXFLAGS = $(CPPUNIT_CFLAGS) $(libusb_CFLAGS)
-OlaClientTester_LDFLAGS = $(CPPUNIT_LIBS) -ldl -lprotobuf
-OlaClientTester_LDADD = @LIBS@ ${PLUGIN_LIBS} \
-                        $(top_builddir)/common/libolacommon.la \
-                        $(top_builddir)/olad/libolaserver.la \
-                        ./libola.la
-
 ## @end 1
 ## @start 2
 # Regenerate common.h with config.status whenever common-h.in changes.
diff --git a/olad/Makefile.am b/olad/Makefile.am
index b4739ea..d25f0d4 100644
--- a/olad/Makefile.am
+++ b/olad/Makefile.am
@@ -45,16 +45,3 @@ olad_CXXFLAGS = -Wall -Wformat -W
 olad_LDFLAGS = -rdynamic -ldl -lprotobuf $(UUID_LIBS)
 olad_LDADD = @LIBS@ libolaserver.la \
              $(top_builddir)/common/libolacommon.la
-
-# Test Programs
-TESTS = OlaTester
-check_PROGRAMS = $(TESTS)
-OlaTester_SOURCES = OlaServerTester.cpp \
-                    UniverseTest.cpp DeviceTest.cpp DeviceManagerTest.cpp \
-                    DmxSourceTest.cpp PluginManagerTest.cpp \
-                    PreferencesTest.cpp PortManagerTest.cpp PortTest.cpp \
-                    OlaServerServiceImplTest.cpp ClientTest.cpp
-OlaTester_CXXFLAGS = $(CPPUNIT_CFLAGS)
-OlaTester_LDFLAGS = $(CPPUNIT_LIBS)
-OlaTester_LDADD = @LIBS@ $(top_builddir)/olad/libolaserver.la \
-                  $(top_builddir)/common/libolacommon.la
diff --git a/plugins/artnet/Makefile.am b/plugins/artnet/Makefile.am
index 1c60843..7204ac8 100644
--- a/plugins/artnet/Makefile.am
+++ b/plugins/artnet/Makefile.am
@@ -11,16 +11,3 @@ libolaartnetnode_la_SOURCES = ArtNetNode.cpp
 lib_LTLIBRARIES = libolaartnet.la
 libolaartnet_la_SOURCES = ArtNetPlugin.cpp ArtNetDevice.cpp ArtNetPort.cpp
 libolaartnet_la_LIBADD = ./libolaartnetnode.la messages/libolaartnetconf.la
-
-# Test programs
-TESTS = ArtNetTester
-check_PROGRAMS = $(TESTS)
-ArtNetTester_SOURCES = ArtNetNodeTest.cpp \
-                       ArtNetTester.cpp \
-                       MockUdpSocket.cpp
-
-ArtNetTester_CXXFLAGS = $(CPPUNIT_CFLAGS)
-ArtNetTester_LDFLAGS = $(CPPUNIT_LIBS)
-ArtNetTester_LDADD = -lprotobuf \
-                     ./libolaartnetnode.la \
-                     ../../common/libolacommon.la
diff --git a/plugins/dummy/Makefile.am b/plugins/dummy/Makefile.am
index 11c5330..a7e81b6 100644
--- a/plugins/dummy/Makefile.am
+++ b/plugins/dummy/Makefile.am
@@ -6,13 +6,3 @@ EXTRA_DIST = DummyPlugin.h DummyDevice.h DummyPort.h
 lib_LTLIBRARIES = liboladummy.la
 liboladummy_la_SOURCES =  DummyPlugin.cpp DummyDevice.cpp DummyPort.cpp
 liboladummy_la_LIBADD = @LIBS@
-
-TESTS = DummyPluginTester
-check_PROGRAMS = $(TESTS)
-DummyPluginTester_SOURCES = DummyPluginTester.cpp DummyPortTest.cpp
-DummyPluginTester_CXXFLAGS = $(CPPUNIT_CFLAGS)
-DummyPluginTester_LDFLAGS = $(CPPUNIT_LIBS) -ldl -lprotobuf
-DummyPluginTester_LDADD = @LIBS@ \
-                          ./liboladummy.la \
-                          $(top_builddir)/common/libolacommon.la \
-                          $(top_builddir)/olad/libolaserver.la
diff --git a/plugins/espnet/Makefile.am b/plugins/espnet/Makefile.am
index 46ef33d..86efcf8 100644
--- a/plugins/espnet/Makefile.am
+++ b/plugins/espnet/Makefile.am
@@ -8,12 +8,3 @@ lib_LTLIBRARIES = libolaespnet.la
 libolaespnet_la_SOURCES = EspNetPlugin.cpp EspNetDevice.cpp EspNetPort.cpp \
                           EspNetNode.cpp RunLengthDecoder.cpp
 libolaespnet_la_LIBADD = @LIBS@
-
-# Test Programs
-TESTS = EspNetTester
-check_PROGRAMS = $(TESTS)
-EspNetTester_SOURCES = EspNetTester.cpp RunLengthDecoderTest.cpp \
-                       RunLengthDecoder.cpp
-EspNetTester_CXXFLAGS = $(CPPUNIT_CFLAGS)
-EspNetTester_LDFLAGS = $(CPPUNIT_LIBS)
-EspNetTester_LDADD = @LIBS@ -lprotobuf ../../common/libolacommon.la
diff --git a/plugins/shownet/Makefile.am b/plugins/shownet/Makefile.am
index 658aec4..e9e4b54 100644
--- a/plugins/shownet/Makefile.am
+++ b/plugins/shownet/Makefile.am
@@ -9,12 +9,3 @@ lib_LTLIBRARIES = libolashownet.la
 libolashownet_la_SOURCES = ShowNetPlugin.cpp ShowNetDevice.cpp ShowNetPort.cpp\
                            ShowNetNode.cpp
 libolashownet_la_LIBADD = @LIBS@
-
-# Test Programs
-TESTS = ShowNetTester
-check_PROGRAMS = $(TESTS)
-ShowNetTester_SOURCES = ShowNetTester.cpp \
-                        ShowNetNode.cpp ShowNetNodeTest.cpp
-ShowNetTester_CXXFLAGS = $(CPPUNIT_CFLAGS)
-ShowNetTester_LDFLAGS = $(CPPUNIT_LIBS)
-ShowNetTester_LDADD = @LIBS@ -lprotobuf ../../common/libolacommon.la
diff --git a/plugins/usbpro/Makefile.am b/plugins/usbpro/Makefile.am
index 2d5f801..31c7411 100644
--- a/plugins/usbpro/Makefile.am
+++ b/plugins/usbpro/Makefile.am
@@ -18,22 +18,3 @@ libolausbpro_la_SOURCES = ArduinoRGBDevice.cpp DmxTriDevice.cpp \
                           DmxterDevice.cpp UsbProDevice.cpp \
                           UsbProPlugin.cpp
 libolausbpro_la_LIBADD = ./libolausbprowidget.la messages/libolausbproconf.la
-
-
-# Test programs
-TESTS = UsbProTester
-check_PROGRAMS = $(TESTS)
-UsbProTester_SOURCES = ArduinoWidgetTest.cpp \
-                       DmxTriWidgetTest.cpp \
-                       DmxterWidgetTest.cpp \
-                       MockUsbWidget.cpp \
-                       UsbProTester.cpp \
-                       UsbProWidgetTest.cpp \
-                       UsbWidgetTest.cpp \
-                       WidgetDetectorTest.cpp
-
-UsbProTester_CXXFLAGS = $(CPPUNIT_CFLAGS)
-UsbProTester_LDFLAGS = $(CPPUNIT_LIBS)
-UsbProTester_LDADD = -lprotobuf \
-                     ./libolausbprowidget.la \
-                     ../../common/libolacommon.la
