diff --git a/olad/Olad.cpp b/olad/Olad.cpp
index 222c1fb..17ad890 100644
--- a/olad/Olad.cpp
+++ b/olad/Olad.cpp
@@ -414,11 +414,6 @@ int main(int argc, char *argv[]) {
   ola::ExportMap export_map;
   Setup(argc, argv, &opts);
 
-  if (!geteuid()) {
-    OLA_FATAL << "Attempting to run as root, aborting.";
-    return -1;
-  }
-
   InitExportMap(&export_map, argc, argv);
 
   if (!InstallSignals())
