diff --git a/olad/Olad.cpp b/olad/Olad.cpp
index af0faf3..92c5c10 100644
--- a/olad/Olad.cpp
+++ b/olad/Olad.cpp
@@ -20,7 +20,6 @@
  *
  */
 
-#include <execinfo.h>
 #include <fcntl.h>
 #include <getopt.h>
 #include <signal.h>
@@ -61,12 +60,7 @@ typedef struct {
  * Print a stack trace on seg fault
  */
 static void sig_segv(int signo) {
-  enum {STACK_SIZE = 64};
-  void *array[STACK_SIZE];
-  size_t size = backtrace(array, STACK_SIZE);
-
   cout << "Recieved SIGSEGV or SIGBUS" << endl;
-  backtrace_symbols_fd(array, size, STDERR_FILENO);
   exit(1);
   (void) signo;
 }
