diff --git a/common.mk b/common.mk
index 91e49af..0393504 100644
--- a/common.mk
+++ b/common.mk
@@ -5,10 +5,5 @@ WARNING_CXXFLAGS = -I$(top_builddir)/include \
 
 COMMON_CXXFLAGS = $(WARNING_CXXFLAGS)
 
-# the genererated protobuf files don't compile with -Werror on win32
-if USING_WIN32_FALSE
-  COMMON_CXXFLAGS += -Werror
-endif
-
 # AM_CXXFLAGS is used when target_CXXFLAGS isn't defined
 AM_CXXFLAGS = $(COMMON_CXXFLAGS)
